dev-requirements.txt

```diff
@@ -1,8 +1,6 @@
 pytest
 pytest-dotenv
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-postgres&subdirectory=plugins/postgres
-git+https://github.com/dbt-labs/dbt-redshift.git
-git+https://github.com/dbt-labs/dbt-snowflake.git
-git+https://github.com/dbt-labs/dbt-bigquery.git
+git+https://github.com/dbt-labs/dbt-core.git@jerco/data-type-macros#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@jerco/data-type-macros#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-bigquery.git@jerco/data-type-macros
```

Comment:
Now that the `dbt-core` PR is merged, only the `dbt-bigquery` PR (https://github.com/dbt-labs/dbt-bigquery/pull/214) is actually blocking this one, since the Redshift + Snowflake PRs don't have substantive changes (just inheriting tests).

TODO: add back imports from `main`
```
git+https://github.com/dbt-labs/dbt-redshift.git
git+https://github.com/dbt-labs/dbt-snowflake.git
```

Reply:
@jtcohen6 Just pushed these in a commit and CI is running now.

Also added back this one along with `dbt-redshift` and `dbt-snowflake`:
- `git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-postgres&subdirectory=plugins/postgres`

Please let me know if I shouldn't have added that one in -- obviously easy to pull it back out.